src/module.ts

```diff
@@ -66,9 +78,18 @@ export default defineNuxtModule<ModuleOptions>({
     nuxt.options.publicRuntimeConfig.algolia = defu(nuxt.options.publicRuntimeConfig.algolia, {
       apiKey: options.apiKey,
       applicationId: options.applicationId,
-      lite: options.lite
+      lite: options.lite,
+      instantSearch: options.instantSearch
     })
 
+    if (options.instantSearch) {
+      nuxt.options.build.transpile.push('vue-instantsearch/vue3')
+
+      if (typeof options.instantSearch === 'object' && options.instantSearch.theme) {
+        nuxt.options.css.push(`instantsearch.css/themes/${options.instantSearch.theme}.css`)
```

Comment:
Maybe add another if statement here that will be used to check if the theme is one of `'reset'|'algolia'|'satellite'`? Because right now, if we pass something else it may not be detected and will result in an error because there won't be appriopriate css file for that name

Reply:
makes sense